.woodpecker.star

```diff
@@ -901,7 +901,7 @@ def localApiTestPipeline(ctx):
                 for storage in params["storages"]:
                     for run_with_remote_php in params["withRemotePhp"]:
                         pipeline = {
-                            "name": "%s-%s%s-%s" % ("CLI" if name.startswith("cli") else "API", name, "-withoutRemotePhp" if not run_with_remote_php else "", storage),
+                            "name": "%s-%s%s-%s" % ("CLI" if name.startswith("cli") else "API", name, "-withoutRemotePhp" if not run_with_remote_php else "", "decomposed" if name.startswith("cli") else storage),
```

Comment:
I'm confused, this will set the name to `decomposed` if the name starts with `cli`? :confused:

Reply:
yes, since we only run CliCommand with decomposed -> I changed cli name

before: 
<img width="380" alt="Screenshot 2025-05-12 at 08 54 44" src="https://github.com/user-attachments/assets/a92cf4ef-3474-454e-8ff8-cd1d83d85efa" />


now: 
<img width="378" alt="Screenshot 2025-05-12 at 08 54 18" src="https://github.com/user-attachments/assets/e337ae01-7006-477b-832f-b0227e9d3d0b" />

Reply:
ah now I understood